Makefile

```diff
@@ -49,7 +49,8 @@ GOCACHE_VOLUME_NAME := stackrox-rox-gocache
 ifneq (,$(findstring -debug,$(shell git rev-parse --abbrev-ref HEAD)))
 	DEBUG_BUILD ?= yes
 endif
-DEBUG_BUILD ?= no
+# DEBUG_BUILD ?= no
+DEBUG_BUILD ?= yes
```

Comment:
why this change?

Reply:
That is just for testing, and must be reverted before merging.